x-pack/plugins/security_solution/public/overview/links.ts

```diff
@@ -109,3 +111,24 @@ export const ecsDataQualityDashboardLinks: LinkItem = {
     }),
   ],
 };
+
+export const mitreAttackCoverageDashboardLinks: LinkItem = {
```

Comment:
The same as above, it looks like it should be `coverageOverviewLinks` to conform with the rest.